run-tests.php

```diff
@@ -3846,6 +3849,7 @@ private function readFile(): void
 
     /**
      * @throws BorkageException
+     * @throws Exception
```

Comment:
I don't see the point in littering the code base with partial exceptions. It's unlikely we'll catch these properly without a tool that verifies them.